server/controllers/reports_proposed/templates/bilan.dot

```diff
@@ -2,143 +2,129 @@
     <!-- asset side -->
-      <table class="balance" border="1">
+      <table class="clean-table" border="1">
         <thead>
           <tr>
-            <th>{{=it.reference}}</th>
-            <th>{{=it.actif}}</th>
+            <th>{{=it.i18nBilan.REFERENCE}}</th>
+            <th>{{=it.i18nBilan.ACTIF}}</th>
             <th colspan="3">Exercice N</th>
             <th>Ex. N-1</th>
           </tr>
 
           <tr>
             <th colspan="2"></th>
-            <th>{{=it.brut}}</th>
-            <th>{{=it.amort_prov}}</th>
-            <th>{{=it.net}}</th>
-            <th>{{=it.net}}</th>
+            <th>{{=it.i18nBilan.BRUT}}</th>
+            <th>{{=it.i18nBilan.AMORT_PROV}}</th>
+            <th>{{=it.i18nBilan.NET}}</th>
+            <th>{{=it.i18nBilan.NET}}</th>
           </tr>
         </thead>
         <tbody>
           {{~it.assetSide : section : ind1}}
             <tr>
               <td></td>
-              <td><b>{{=section.sectionBilanLabel}}({{=section.sectionBilanPosition}})</b></td>
+              <td class="text-center"><b>{{=section.sectionBilanLabel}}({{=section.sectionBilanPosition}})</b></td>
               <td colspan="3"></td>
               <td></td>
             </tr>
               {{~section.grefs : gref : ind2}}
                 <tr>
-                  <td><b>{{=gref.greferenceAbbr}}</b></td>
+                  <td class="text-center"><b>{{=gref.greferenceAbbr}}</b></td>
                   <td><b>{{=gref.greferenceLabel}}</b></td>
                   <td colspan="3"></td>
                   <td></td>
                 </tr>
                   {{~gref.refs : ref : ind3}}
                     <tr>
-                      <td>{{=ref.referenceAbbr}}</td>
+                      <td class="text-center">{{=ref.referenceAbbr}}</td>
                       <td>{{=ref.referenceLabel}}</td>
-                      <td>{{=ref.brut_view}}</td>
-                      <td>{{=ref.amort_prov_view}}</td>
-                      <td>{{=ref.net_view}}</td>
-                      <td>{{=ref.previousNet_view}}</td>
+                      <td class="text-center">{{=ref.brut_view}}</td>
+                      <td class="text-center">{{=ref.amort_prov_view}}</td>
+                      <td class="text-center">{{=ref.net_view}}</td>
+                      <td class="text-center">{{=ref.previousNet_view}}</td>
                     </tr>
                   {{~}}
               {{~}}
               <tr>
                 <td></td>
-                <td><b>{{=it.total}} {{=section.sectionBilanLabel}} ({{=section.sectionBilanPosition}})</b></td>
-                <td><b>{{=section.totalBrut_view}}</b></td>
-                <td><b>{{=section.totalAmortProv_view}}</b></td>
-                <td><b>{{=section.totalNet_view}}</b></td>
-                <td><b>{{=section.totalPreviousNet_view}}</b></td>
+                <td class="text-center"><b>{{=it.i18nBilan.TOTAL}} {{=section.sectionBilanLabel}} ({{=section.sectionBilanPosition}})</b></td>
```

Comment:
Did you try using a single "text-center" class on the `<tr>` element?  It might be clearer.

If you are using `</td><b></b></td>`, you probably mean `<th></th>`.  Why not use that instead?